### PR TITLE
feat(react-native): Xcode plugin includes collect modules script

### PR DESCRIPTION
**Pull request by krystofwoldrich**
The same as https://github.com/getsentry/sentry-wizard/pull/210

For RN < 0.69

### CODE REVIEW

**Comment by marandaneto** [comment]
I'd rather call `patch/v1.4` instead of `1.x` branch as we talked about before.
You'd need to add the changelog later back into `master` as well since you can't merge the branch into `master`.

**Comment by marandaneto** [comment]
Similar comment https://github.com/getsentry/sentry-wizard/pull/210#issuecomment-1329097376
If this is tested, LGTM.

File: .github/workflows/codeql-analysis.yml
Comment (marandaneto) [inline .github/workflows/codeql-analysis.yml:16]: If you make a draft PR from `1.x` to `master` (after merging), all the workflows would run and this is not necessary.
So you can either keep this and later remove the 1.x from workflows, or merge without adding the 1.x workflow but open a draft PR to check that everything is green.

**Review by marandaneto — APPROVED** [review]

**Comment by krystofwoldrich** [comment]
It's tested both the v1 and v2.

**Comment by jakobsen9** [comment]
Hi,

The added collect modules script fails my build on RN 0.64.4. I get an error saying "@sentry/react-native/scripts/collect-modules.sh: No such file or directory". Any ideas why it fails? It seems above that you tested properly?

**Pull request merged**
